lib/SitesManager.php

```diff
@@ -151,8 +151,8 @@ public function getSitesToDisplay() {
 			}
 
 			$site['url'] = str_replace(
-				['{email}', '{uid}', '{displayname}', '{jwt}'],
-				array_map('rawurlencode', [$email, $uid, $displayName, $jwt]),
+				['{email}', '{uid}', '{displayname}', '{jwt}', '{groups}'],
+				array_map('rawurlencode', [$email, $uid, $displayName, $jwt, implode(",", $groups)]),
```

Comment:
I'm afraid that this is not configurable enough. Maybe someone wants to have the groups separate by dashes or semicolons or something else.